src/components/Atoms/Image/Icon/Icon.stories.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
Is it possible to use TypeScript files for this `*.stories.js` files? Then the TypeScript compiler will let you know when made (conflicting) changes to the components used in these files.

Reply:
as discussed